Source/WebGPU/WebGPU/Device.mm

```diff
@@ -151,6 +153,9 @@ static void captureFrame(id<MTLDevice> captureObject)
 #endif
 #endif
 
+#if COREVIDEO_SUPPORTS_METAL
```

Comment:
When would this ever be false? Doesn't Core Video always support Metal on all our platforms?

Reply:
watchOS

Reply:
(discussed offline, resolving conversation)